cmd/autothrottle/main.go

```diff
@@ -206,7 +207,7 @@ func main() {
 	var ticker = time.NewTicker(time.Duration(Config.Interval) * time.Second)
 
 	// TODO(jamie): refactor this loop.
-	for ; ; <-ticker.C {
+	for {
 		interval++
```

Comment:
nit: based on how it's used in this function, I think we should only be doing `interval++` in the case when we receive from `ticker.C`.